scripts/build.py

```diff
@@ -486,10 +629,12 @@ def filter_hashes(file_hashes):
     and prefixes / in front of the keys.
```

Comment:
This was there before, but could you put quotation marks around the "/"? I got a bit confused when reading this.

Reply:
Done.